instantlaunches/main.go

```diff
@@ -114,11 +114,9 @@ type InstantLaunchMapping map[string]*InstantLaunchSelector
 func (i InstantLaunchMapping) Scan(value interface{}) error {
 	switch v := value.(type) {
 	case []byte:
-		json.Unmarshal(v, &i)
-		return nil
+		return json.Unmarshal(v, &i)
```

Comment:
I wasn't sure if this was swallowing/ignoring the error deliberately, in which case this would be a wrong fix (and below)

Reply:
Weird. I vaguely remember intending to swallow the error, but I'm not certain why. Kicking myself for not commenting it :/

Reply:
It feels wrong, but I'd recommend swallowing the error for now. This might take some more poking at to figure out how to fix it correctly.